package.json

```diff
@@ -34,32 +34,32 @@
     "tslint": "tslint --fix --project .",
     "prettier": "prettier --write lib/*.ts test/*.js",
     "prepare": "tsc",
-    "test": "tap --no-ts --node-arg=-r --node-arg=ts-node/register --no-coverage test/*_test.js"
```

Comment:
`--no-ts` is no longer a valid param of `tap`. `--no-coverage` was replaced by `--disable-coverage`